src/components/ContributorTasks/ContributorTasks.test.tsx

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import format from 'date-fns/format';
+
+import ContributorTasks, {ContributorTasksProps} from './index';
+
+describe('ContributorTasks', () => {
+  const props: ContributorTasksProps = {
+    className: 'test',
+    tasks: [
+      {
+        amount_paid: '2800',
+        completed_by: 'contributor 1',
+        completed_date: new Date(Date.now()),
+        issue_id: '111',
+        pr_id: '222',
+        repository: 'Kotlin-SDK',
+        title: 'Sofware engineer',
+      },
+      {
+        amount_paid: '2800',
+        completed_by: 'contributor 2',
+        completed_date: new Date(Date.now()),
+        issue_id: '333',
+        pr_id: '444',
+        repository: 'Website',
+        title: 'Front End Developer',
+      },
+    ],
+  };
+
+  it('renders without crashing', () => {
+    render(<ContributorTasks {...props} />);
+
+    expect(screen.getByTestId('ContributorTasks')).toBeTruthy();
+  });
+
+  it('renders with default className', () => {
+    render(<ContributorTasks {...props} />);
+    const taskComp = screen.getByTestId('ContributorTasks');
+
+    expect(taskComp).toHaveClass('ContributorTasks');
```

Comment:
can you add more `expect`s here? test for the other DOM elements that you gave `data-testid` to (if they have a default className)